openreview/conference/templates/reviewRebuttalProcess.js

```diff
@@ -0,0 +1,86 @@
+function(){
+    var or3client = lib.or3client;
+
+    var CONFERENCE_ID = '';
+    var SHORT_PHRASE = '';
+    var AUTHORS_NAME = '';
+    var PROGRAM_CHAIRS_ID = '';
+    var USE_AREA_CHAIRS = false;
+
+    var forumNote = or3client.or3request(or3client.notesUrl+'?id='+note.forum, {}, 'GET', token);
+    var reviewNote = or3client.or3request(or3client.notesUrl+'?id='+note.replyto, {}, 'GET', token);
+
+    Promise.all([forumNote, reviewNote]).then(function([result1, result2]) {
+      var forum = result1.notes[0];
+      var review = result2.notes[0];
+      var promises = [];
+
+      var AUTHORS_ID = CONFERENCE_ID + '/Paper' + forum.number + '/' + AUTHORS_NAME;
+      //TODO: use the variable instead, when we have anonymous groups integrated
+      var REVIEWERS_ID = CONFERENCE_ID + '/Paper' + forum.number + '/Reviewers';
+      var AREA_CHAIRS_ID = CONFERENCE_ID + '/Paper' + forum.number + '/Area_Chairs';
+      var AREA_CHAIR_1_ID = CONFERENCE_ID + '/Paper' + forum.number + '/Area_Chair1';
+      var ignoreGroups = note.nonreaders || [];
+      ignoreGroups.push(note.tauthor);
+
+      var content = 'To view the rebuttal, click here: ' + baseUrl + '/forum?id=' + note.forum + '&noteId=' + note.id;
+
+      if (PROGRAM_CHAIRS_ID) {
+        var program_chair_mail = {
+          groups: [PROGRAM_CHAIRS_ID],
+          ignoreGroups: ignoreGroups,
+          subject: '[' + SHORT_PHRASE + '] A rebuttal has been received on Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+          message: 'We have received a review rebuttal on a submission to ' + SHORT_PHRASE + '.\n\n' + content
+        };
+        promises.push(or3client.or3request( or3client.mailUrl, program_chair_mail, 'POST', token ));
+      }
+
+      var authors_writer_mail = {
+        groups: note.signatures,
+        subject: '[' + SHORT_PHRASE + '] Your rebuttal has been received on your submission - Paper number: ' + forum.number + ', Paper title: "' + forum.content.title + '"',
+        message: 'We have received your rebuttal on your submission to ' + SHORT_PHRASE + '.\n\nPaper number: ' + forum.number + '\n\nPaper title: ' + forum.content.title + '\n\n' + content
```

Comment:
"Your review rebuttal to the ${short_phrase} submission titled, "${title}" has been posted."

Reply:
same here